wdl/TasksClusterBatch.wdl

```diff
@@ -0,0 +1,504 @@
+                                   mem_gb: 3.75,
+                                   disk_gb: ceil(10 + size(vcf, "GB") * 2),
+                                   boot_disk_gb: 10,
+                                   preemptible_tries: 3,
+                                   max_retries: 1
+                               }
+    RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+    output {
+        File out = "~{output_prefix}.vcf.gz"
+        File out_index = "~{output_prefix}.vcf.gz.tbi"
+    }
+    command <<<
+        set -euo pipefail
+        cut -f1,2 ~{reference_fasta_fai} > genome.file
+        bcftools query -f '%CHROM\t%POS\t%POS\t%ID\t%SVTYPE\n%CHROM\t%END\t%END\t%ID\t%SVTYPE\n%CHR2\t%END2\t%END2\t%ID\t%SVTYPE\n' ~{vcf} \
+            | awk '$1!="."' \
+            | sort -k1,1V -k2,2n -k3,3n \
+            > ends.bed
+        bedtools intersect -sorted -u -wa -g genome.file -wa -a ends.bed -b ~{intervals} | cut -f4 | sort | uniq \
+            > excluded_vids.list
+        bcftools view -i '%ID!=@excluded_vids.list' ~{vcf} -Oz -o ~{output_prefix}.vcf.gz
+        tabix ~{output_prefix}.vcf.gz
+    >>>
+    runtime {
+        cpu: select_first([runtime_attr.cpu_cores, default_attr.cpu_cores])
+        memory: select_first([runtime_attr.mem_gb, default_attr.mem_gb]) + " GiB"
+        disks: "local-disk " + select_first([runtime_attr.disk_gb, default_attr.disk_gb]) + " HDD"
+        bootDiskSizeGb: select_first([runtime_attr.boot_disk_gb, default_attr.boot_disk_gb])
+        docker: sv_base_mini_docker
+        preemptible: select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
+        maxRetries: select_first([runtime_attr.max_retries, default_attr.max_retries])
+    }
+}
+
+task ExcludeIntervalsByIntervalOverlap {
+    input {
+        File vcf
+        Float overlap_fraction
+        File intervals
+        File intervals_index
+        File reference_fasta_fai
+        String output_prefix
+        String sv_base_mini_docker
+        RuntimeAttr? runtime_attr_override
+    }
+
+    RuntimeAttr default_attr = object {
+                                   cpu_cores: 1,
+                                   mem_gb: 3.75,
+                                   disk_gb: ceil(10 + size(vcf, "GB") * 2),
+                                   boot_disk_gb: 10,
+                                   preemptible_tries: 3,
+                                   max_retries: 1
+                               }
+    RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+
+    output {
+        File out = "~{output_prefix}.vcf.gz"
+        File out_index = "~{output_prefix}.vcf.gz.tbi"
+    }
+    command <<<
+        set -euo pipefail
+        cut -f1,2 ~{reference_fasta_fai} > genome.file
+        bcftools query -f '%CHROM\t%POS\t%END\t%ID\t%SVTYPE\n' ~{vcf} > variants.bed
+        bedtools coverage -sorted -g genome.file -f ~{overlap_fraction} -a variants.bed -b ~{intervals} \
```

Comment:
I was going to suggest that bedtools coverage might work on the vcf directly without converting to bed first but then I decided to double check and found that it uses SVLEN to determine the end of the interval.. so best not to mess with that I think.